### ISSUE TITLE
Crazy render breakdown, unplayable [BUG]

### ISSUE THREAD

**Issue by mrbuilder1961** [haubna/PhysicsMod#265]
**Describe the bug**
Whenever you kill a mob with any enchanted items and a death setting (not off) it will completely crash your view. This only happens when you can see the corpse, right when you kill a mob, or if you break a spawner with any mob inside that has any enchanted items equipped. It also doesn't seem to cause this issue if you change the mob settings to not ragdoll/off, or particles for blocks.

**To Reproduce**
1. Find a mob with any enchanted item equipped that is visible to the player
2. Kill it
3. Screen will break
**OR**
1. Find a monster spawner that has a mob with visible enchanted gear
2. Break it
3. Screen will break

**Videos**
See [this](https://vimeo.com/669017101), [this](https://vimeo.com/669017072), and also [this](https://vimeo.com/669017048).

**Notes**
The only way to fix is to restart the game or disable mob physics entirely.
In the log, the error line (for the skeleton) is:

```
[22Jan2022 16:34:09.756] [Render thread/INFO] [STDERR/]: [net.diebuddies.physics.PhysicsMod:blockifyEntity:538]: error rendering class net.minecraft.entity.monster.SkeletonEntity
[22Jan2022 16:34:09.756] [Render thread/INFO] [STDERR/]: [net.diebuddies.physics.PhysicsMod:blockifyEntity:539]: java.lang.IllegalArgumentException: Duplicate delegates
```
(see [pastebin](https://pastebin.com/0jEKbZM9) for entire stack trace)

I don't believe any other of my mods are causing this (aside from spawning the powerful mobs lol), but here's a list anyway because modding is weird sometimes:
![image](https://user-images.githubusercontent.com/66807297/150658727-b66c4ae8-32f6-4a19-b066-7dbd86f00024.png)

**Specs**
Minecraft Version: Forge 1.16.5
Physics Mod Version: 2.6.5
OS: Windows 10.0.19044
CPU: Intel i9-11900k @ 3.5GHz, 8 Cores & 16 Logical Processors (Intel UHD 750 Integrated Graphics)
RAM: 32GB (2x16GB) @ 2666MHz, overclocked to 3500MHz
Storage: Samsung SSD 970 Evo Plus 500GB

If any more information is needed, I'll be happy to provide it if possible.

**Comment by haubna** [comment]
Thank you so much for the detailed report! It's rare to see so much information. I've been looking for a way to reproduce this bug the last 2 days. This will make it easy to fix it. Fix will be included in 2.6.6!

**Comment by mrbuilder1961** [comment]
No problem!! I really want a fix for this, but as a fellow mod dev I know how bugs can be weird and nit-picky. Good luck!

**Comment by haubna** [comment]
Released 2.6.6. Should be fixed! Thanks for the report :)
